src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -383,6 +384,12 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                 }, DatabaseShutdown);
                 _serverStore.LicenseManager.LicenseChanged += LoadTimeSeriesPolicyRunnerConfigurations;
                 IoChanges.OnIoChange += CheckWriteRateAndNotifyIfNecessary;
+
+                using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext ctx))
```

Comment:
let's do this before `_clusterTransactionsTask`

Reply:
This should be part of the document's database context pool, not the server one

Reply:
let's have a test that we able to set it properly.

Reply:
Done